### DISCUSSION TITLE
Separate Skin from Layout / Having Layout Presets

### DISCUSSION THREAD

**Discussion by rvnexe** [ppy/osu#27901] [Ideas]
Hi. There are some people who have moved from stable to lazer, and still, they're using their custom skins.

The layout editor is a huge feature that lets you edit the HUD, but it's attached to the Skin. 
For example, if you want to change the skin, while maintaining your custom layout, you'll have to Export the SkinA, Extract the MainHUDComponents.json, Add it to the SkinB osk file, Import it.

What if we could change the layout without having to go this complicated way? (look at this pic that I made)
![layout-selection-concept-osu](https://github.com/ppy/osu/assets/89677246/cdf05af4-0f12-46cc-8726-230b80549885)

**Comment by marioman63** [comment]
This would also be great for those of us who like playing with map skins enabled. I love the argon progress bar, but you likely won't see it in any map skins due to them most certainly being made outside of lazer.
